keel-web/src/main/kotlin/com/netflix/spinnaker/keel/rest/ApplicationController.kt

```diff
@@ -228,4 +229,14 @@ class ApplicationController(
   ) {
     applicationService.deleteVeto(application, targetEnvironment, reference, version)
   }
+
+  @GetMapping(
+    path = ["/{application}/events"],
```

Comment:
I realized we didn't have an endpoint for this and it helps at the very least with testing.